File: app/src/main/java/com/example/toshiba/parkme/MainActivity.java
package com.example.toshiba.parkme;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.view.View.OnClickListener;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // Set the content of the activity to use the activity_main.xml layout file
        setContentView(R.layout.activity_main);

        Button button1 = (Button) findViewById(R.id.signup);

        button1.setOnClickListener((new OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent button1Intent = new Intent(MainActivity.this, SignupActivity.class);

                startActivity(button1Intent);
            }
        }));

        Button button2 = (Button) findViewById(R.id.login);

        button2.setOnClickListener((new OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent button2Intent = new Intent(MainActivity.this, LoginActivity.class);

                startActivity(button2Intent);
            }
        }));
    }
}

File: app/src/main/java/com/example/toshiba/parkme/activity/MainActivity.kt
package com.example.toshiba.parkme.activity

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.example.toshiba.parkme.R


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val button1 : Button = findViewById(R.id.signup)
        button1.setOnClickListener {
            val button1Intent = Intent(this@MainActivity, SignUpActivity::class.java)
            startActivity(button1Intent)
            finish()
        }
        val button2 : Button = findViewById(R.id.login)
        button2.setOnClickListener {
            val button2Intent = Intent(this@MainActivity, LoginActivity::class.java)
            startActivity(button2Intent)
            finish()
        }
    }

    override fun onBackPressed() {
        finish()
    }
}

